src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3252,6 +3252,8 @@ private BoundExpression BindArgumentExpression(BindingDiagnosticBag diagnostics,
 
             // Parameter types should be taken from the least overridden member:
             var parameters = methodResult.LeastOverriddenMember.GetParameters();
+            // But ref kinds should be taken from the actually called member:
```

Comment:
>// But ref kinds should be taken from the actually called member: [](http://example.com/codeflow?start=12&length=65)

Why is that? My initial reaction is that we shouldn't be making a change like that. Intermediate overrides shouldn't matter at the call site, the initial virtual declaration matters.